src/main/java/nextstep/step3/RacingCarController.java

```diff
@@ -0,0 +1,36 @@
+package nextstep.step3;
+
+import nextstep.step3.domain.RacingCars;
+import nextstep.step3.domain.RandomGenerator;
+import nextstep.step3.view.ResultView;
+
+public class RacingCarController {
+	private static final String TRY_NUMBER_MUST_POSITIVE = "시도 횟수는 양수이어야 합니다.";
+
+	public void startGame(String carName, int tryNumber) {
+		validate(tryNumber);
+		RacingCars racingCars = new RacingCars(carName);
+		tryRacing(tryNumber, racingCars);
+	}
+
+	protected void validate(int tryNumber) {
+		if (tryNumber < 0) {
+			throw new IllegalArgumentException(TRY_NUMBER_MUST_POSITIVE);
+		}
+	}
+
+	private void tryRacing(int tryNumber, RacingCars racingCars) {
+		System.out.println("실행 결과");
+		ResultView resultView = new ResultView();
+		for(int i = 0; i < tryNumber; i++) {
+			racingCarMove(racingCars);
```

Comment:
RacingCars에게 움직이라고 메세지를 보내도록 구현하면 어떤 장점이 있을까요 :) ?
```suggestion
			racingCars.move(randomGenerator);
```